queries.sql

```diff
@@ -7,3 +7,28 @@ SELECT name, neutered FROM animals WHERE neutered = 'TRUE';
 SELECT name FROM animals WHERE NOT name = 'Gabumon';
 SELECT name, weight_kg FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```

Comment:
**Optional**

- [ ] Please update the table as a  count, its not that is not working but for more understanding purposes.